pep-0636.rst

```diff
@@ -526,23 +557,6 @@ Several other key features:
           case Color.BLUE:
               print("I'm feeling the blues :(")
 
-- The literals ``None``, ``False`` and ``True`` are treated specially:
-  comparisons to the subject are done using ``is``.  This::
-
-      match b:
-          case True:
-              print("Yes!")
-
-  is exactly equivalent to this::
-
-      if b is True:
-          print("Yes!")
```

Comment:
I feel that the special case for True/False/None is at least worth a brief mention. Maybe without example?